### DISCUSSION TITLE
Pair coding

### DISCUSSION THREAD

**Discussion by Blackie360** [community/community#124661] [Other Feature Feedback, Questions, & Ideas]
### Body

I created a pair Programming "Code buddy " platform were a user can sign  up using Google email account or GitHub account .
a user can create a room where other users can join it and code together . 
link: https://pair-coding2-0.vercel.app/

FEATURES: 
1. social Auth; Google and GitHub
<img width="468" alt="auth" src="https://github.com/community/community/assets/71150929/712a92de-9c93-4d24-a37f-67206a23e431">

2. Rooms are public one can have random developers  join in to exchange ideas and debug skills
<img width="895" alt="createroom" src="https://github.com/community/community/assets/71150929/1e5c8042-1330-4083-8c64-8eb964eb672a">
create Room Form
GitHub repo link can be added
<img width="877" alt="creatroom" src="https://github.com/community/community/assets/71150929/e3d686ed-2b91-4d10-a3cd-8ee2f0499102">

4. Share screen ,audio, webcam capabilities  and many more
<img width="677" alt="paircoding" src="https://github.com/community/community/assets/71150929/c86ae8c2-f31a-4458-8c2f-dd58a3e373e7">
Screen sharing 💯 
<img width="627" alt="shared" src="https://github.com/community/community/assets/71150929/ca150aaf-1cdb-4471-83b4-1493197f988f">

5. A user can block unwanted users from joining there pair coding session 
Tech stack : Next js , Stream.io , postgress db , Supabase, Nextauth

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by CopperEagle** [comment]
That looks cool!  Pair coding is fun :tada: Thank you for sharing

What could be interesting to try out is to have both share the same editor and both could do changes, seeing what text the other person highlighted, etc. Could be an option next to screen sharing, etc.

*Ps. This category's [guidelines](https://github.com/orgs/community/discussions/51384) basically ban self-promotion. Instead of having this in the "programming help" category, you maybe could have this in the "General" category, with the label "Show & Tell".*

**Reply by Blackie360** [reply]
thanks for the feedback 
sorry about that let me move it  to the category you have suggested
